packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -904,28 +901,49 @@ abstract class TableBase extends Resource implements ITable {
     }).attachTo(this);
   }
 
+  /**
+   * How many requests are throttled on this table.
+   *
+   * This will sum errors across all possible operations.
+   * Note that by default, each individual metric will be calculated as a sum over a period of 5 minutes.
+   * You can customize this by using the `statistic` and `period` properties.
+   */
+  public metricThrottledRequestsForOperation(props?: OperationsMetricOptions): cloudwatch.IMetric {
```

Comment:
We can't change the contract for this method. We could add a new one
`metricThrottledRequestsForOperations` (plural).